### PR TITLE
Added coverage to ban and disconnect peer

### PR DESCRIPTION
**Pull request by majikandy**
Added coverage over existing method that was missing the coverage

### CODE REVIEW

File: src/Stratis.Bitcoin.Features.Consensus.Tests/PeerBanningTest.cs [hunk 1]
```diff
@@ -66,6 +66,30 @@ private async Task<(TestChainContext context, IPEndPoint peerEndPoint)> Initiali
             Assert.True(context.PeerBanning.IsBanned(peerEndPoint));
         }
 
+        [Fact]
+        public async Task NodeIsSynced_PeerSendsAMutatedBlockAndPeerDisconnected_AndAddressIsNull_ThePeerGetsBanned_Async()
+        {
+            await this.NodeIsSynced_PeerSendsABadBlockAndPeerDisconnectedAndAddressIsNull_ThePeerGetsBanned_Async(
```
Comment (codingupastorm) [inline src/Stratis.Bitcoin.Features.Consensus.Tests/PeerBanningTest.cs]: This looks weird now but I'm guessing you may have more tests in future using the same method?

**Comment by fassadlr** [comment]
@majikandy does this PR relate to #1423 ?

File: src/Stratis.Bitcoin.Features.Consensus.Tests/PeerBanningTest.cs (hunk 1 above)
Comment (majikandy) [reply src/Stratis.Bitcoin.Features.Consensus.Tests/PeerBanningTest.cs]: Yeah it does, just copied the way another test was done in the file even though not varying the way the blocks were bad for this one, it was easier

**Comment by majikandy** [comment]
Yes @fassadlr but doesn't close it

**Pull request merged**